keywords.robot

```diff
@@ -546,6 +547,21 @@ Prepare Test Suite
     END
     IF    '${CONFIG}' == 'rpi-3b'    Verify Number Of Connected SD Wire Devices
 
+Import Osfv Libraries
+    [Documentation]    Import osfv_cli libraries based on config and command
+    ...    line variables
+    IF    '${SNIPEIT}' == 'yes'
+        Import Library    osfv.rf.snipeit_robot
+        Import Library    osfv.rf.rte_robot.RobotRTE    ${RTE_IP}    True
+    ELSE IF    'sonoff' == '${POWER_CTRL}'
+        Variable Should Exist    ${SONOFF_IP}
+        Import Library    osfv.rf.rte_robot.RobotRTE    ${RTE_IP}    False
+        ...    ${SONOFF_IP}    ${CONFIG}
```

Comment:
What is CONFIG? Can we document this here?

Reply:
271316b396eccc0c67738c27e883e9bbf6eeeb54